hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -687,7 +688,8 @@
             banks: 2,
             pages_per_bank: 256,
             program_resolution: 8, // maximum number of flash words allowed to program at one time
-          }
+          },
+          addr_space: "hart"
```

Comment:
This doesn't look right to me. This memory section isn't describing addresses--It describes properties of the named "mem" region above, which already describes the connected address spaces in the `base_addrs` dictionary.

Instead of specifying the addr_space here, can we do a lookup in the relevant generators to get the needed info?

Reply:
That makes totally sense. I removed the new entry and use the base address field to determine the address space the memory is in ,